easybuild/easyconfigs/g/graph-tool/graph-tool-2.55-foss-2022a.eb

```diff
@@ -0,0 +1,57 @@
+# Author J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'ConfigureMake'
+
+name = 'graph-tool'
+version = '2.55'
+
+homepage = 'https://graph-tool.skewed.de/'
+description = """Graph-tool is an efficient Python module for manipulation and
+ statistical analysis of graphs (a.k.a. networks). Contrary to
+ most other python modules with similar functionality, the core
+ data structures and algorithms are implemented in C++, making
+ extensive use of template metaprogramming, based heavily on
+ the Boost Graph Library. This confers it a level of
+ performance that is comparable (both in memory usage and
+ computation time) to that of a pure C/C++ library."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+toolchainopts = {'cstd': 'c++17'}
+
+source_urls = ['https://downloads.skewed.de/%(name)s/']
+sources = [SOURCE_TAR_BZ2]
+checksums = ['75728e55b774af734954cd48f3d80ba64cf08864ef6143408f3a7422474834b4']
+
+builddependencies = [
+    ('gawk', '5.1.1'),
+    ('pkgconf', '1.8.0'),
```

Comment:
Use pkg-config instead

Reply:
Why? We switched to `pkgconf` for 2022a easyconfigs.

Reply:
Sorry, got them mixed up (again)